OtripleS.Web.Api.Tests.Unit/Services/CourseAttachments/CourseAttachmentServiceTests.Exceptions.RetrieveById.cs

```diff
@@ -48,9 +48,9 @@ public async Task ShouldThrowDependencyExceptionOnRetrieveWhenSqlExceptionOccurs
                 broker.LogCritical(It.Is(SameExceptionAs(expectedCourseAttachmentDependencyException))),
                     Times.Once);
 
-            this.dateTimeBrokerMock.VerifyNoOtherCalls();
             this.loggingBrokerMock.VerifyNoOtherCalls();
```

Comment:
The order here is still.off. 
We need to order those in the *same order* they are called in.
Do we call logging before storage?